homework8.1/task2.js

```diff
@@ -0,0 +1,5 @@
+const array = [1, 2, 3, 4, 5];
+const doubledNumbers = array.map(item => item * 3);
```

Comment:
А де помножити на значення індексу?
```suggestion
const doubledNumbers = array.map((item, index) => item * index);
```